specification/synapse/data-plane/Microsoft.Synapse/preview/2021-07-01-preview/artifacts.json

```
@@ -0,0 +1,5064 @@
+        "featureName": {
+          "type": "string",
+          "description": "The feature name.",
+          "readOnly": true
+        },
+        "value": {
+          "type": "string",
+          "description": "The feature value.",
+          "readOnly": true
+        }
+      }
+    },
+    "SynapseNotebookReference": {
+      "description": "Synapse notebook reference type.",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "Synapse notebook reference type.",
+          "enum": [
+            "NotebookReference"
+          ],
+          "x-ms-enum": {
+            "name": "NotebookReferenceType",
+            "modelAsString": true
+          }
+        },
+        "referenceName": {
+          "type": "string",
+          "description": "Reference notebook name."
+        }
+      },
+      "required": [
+        "type",
+        "referenceName"
+      ]
+    },
+    "SynapseSparkJobReference": {
+      "description": "Synapse spark job reference type.",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "Synapse spark job reference type.",
+          "enum": [
+            "SparkJobDefinitionReference"
+          ],
+          "x-ms-enum": {
+            "name": "SparkJobReferenceType",
+            "modelAsString": true
+          }
+        },
+        "referenceName": {
+          "type": "string",
+          "description": "Reference spark job name."
+        }
+      },
+      "required": [
+        "type",
+        "referenceName"
+      ]
+    },
+    "SqlPoolReference": {
+      "description": "SQL pool reference type.",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "SQL pool reference type.",
+          "enum": [
+            "SqlPoolReference"
+          ],
+          "x-ms-enum": {
+            "name": "SqlPoolReferenceType",
+            "modelAsString": true
+          }
+        },
+        "referenceName": {
+          "type": "string",
+          "description": "Reference SQL pool name."
+        }
+      },
+      "required": [
+        "type",
+        "referenceName"
+      ]
+    },
+    "ArtifactRenameRequest": {
+      "description": "Request body structure for rename artifact.",
+      "type": "object",
+      "properties": {
+        "newName": {
+          "description": "New name of the artifact.",
+          "type": "string",
+          "pattern": "^[A-Za-z0-9_][^<>*#.%&:\\\\+?/]*$",
+          "minLength": 1,
+          "maxLength": 260
+        }
+      }
+    },
+    "KqlScripts_ResourceCollectionResponse": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/KqlScriptResource"
+          }
+        },
+        "nextLink": {
+          "type": "string"
+        }
+      }
```

The `value` property should be required.
```suggestion
      },

      "required": [
        "value"
      ],
```

---

@epkalyanr we can do as suggested but please remove the empty line with line number 4849.